packages/serverless-orchestration/src/ServerlessSpoke.js

```diff
@@ -54,17 +57,28 @@ spoke.post("/", async (req, res) => {
     if (execResponse.error) {
       throw execResponse;
     }
+    // Log the full execResponse in this log. This enables you to retrieve the full output of the child process.
```

Comment:
this is functionality different: the `Process exited with no error` log now contains the _complete_ `execResponse` log. this contains the _full_, non-redacted spoke output. this should make finding logs and their outputs easier within gcp.